lib/tasks/backy_tasks.rake

```diff
@@ -0,0 +1,84 @@
+require "aws-sdk-s3"
+
+namespace :backy do
+  task dump: :environment do
+    pg_dump
+  end
+
+  task push: :environment do
+    file_name = pg_dump
+    save_to_s3(file_name)
+    File.delete(file_name) if File.exist?(file_name)
+  end
+
+  task upload: :environment do
+    file_name = ENV["DUMP_FILE"]
+
+    if file_name.nil? || !File.exist?(file_name)
+      puts "Plese set DUMP_FILE env variable to the existing local file to save to S3"
+
+      exit
+    end
+
+    save_to_s3(file_name)
+  end
+
+  task download: :environment do
+    file_name = ENV["DUMP_FILE"]
+
+    if file_name.nil? || File.exist?(file_name)
+      puts "Plese set DUMP_FILE env variable to the missing remote file to download"
+
+      exit
+    end
+
+    load_from_s3_if_missing(file_name)
+  end
+
+  task list: :environment do
+    list = DB::Backup::List.call
```

Comment:
damn! i even tested this in flytaxien, but because we have the old namespaces there already, i got fooled into thinking things were OK. Nicely spotted though!